src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -17,42 +17,28 @@
 using System;
 using System.Diagnostics;
 using System.Linq;
-using System.Text.Json;
-using System.Text.Json.Serialization;
+#if NET461 || NETSTANDARD2_0
+using OpenTelemetry.Logs;
+#endif
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.Exporter
 {
-    public class ConsoleExporter : BaseExporter<Activity>
+    public class ConsoleExporter<T> : BaseExporter<T>
+        where T : class
     {
-        private readonly JsonSerializerOptions serializerOptions;
-        private readonly bool displayAsJson;
-
         public ConsoleExporter(ConsoleExporterOptions options)
         {
-            this.serializerOptions = new JsonSerializerOptions()
-            {
-                WriteIndented = true,
-            };
-
-            this.displayAsJson = options?.DisplayAsJson ?? false;
-
-            this.serializerOptions.Converters.Add(new JsonStringEnumConverter());
-            this.serializerOptions.Converters.Add(new ActivitySpanIdConverter());
-            this.serializerOptions.Converters.Add(new ActivityTraceIdConverter());
         }
 
-        public override ExportResult Export(in Batch<Activity> batch)
+        public override ExportResult Export(in Batch<T> batch)
         {
-            foreach (var activity in batch)
+            if (typeof(T) == typeof(Activity))
```

Comment:
nit: Would prob be more efficient to check the type in ctor and then set a field instead of checking every batch.